apps/solution/src/components/people/people.module.scss

```diff
@@ -1,3 +1,14 @@
 .container {
   margin: 2rem;
 }
+
+.searchBar {
+  all: revert;
```

Comment:
I don't have a chance to test this code yet, but I'm wondering why this is important ? 
I think we don't have that much CSS code yet. 
Could you please let me know about this ? Thanks